module/bdev/lvol/vbdev_lvol.c

```diff
@@ -1586,7 +1586,7 @@ _vbdev_lvol_set_xattr_cb(void *cb_arg, int lvolerrno)
 	struct spdk_lvol_req *req = cb_arg;
 
 	if (lvolerrno != 0) {
-		SPDK_ERRLOG("Setting xattr failed\n");
+		SPDK_ERRLOG("CB function for setting xattr receive error no: %d.\n", lvolerrno);
```

Comment:
The macro `SPDK_ERRLOG` prints:
- ERROR string
- file name
- row number
- function name

So I think it is not necessary to explicit `CB function for setting xattr`, you could simply modify the message in `Setting xattr failed: error %d\n`. It will be expanded in: 
`vbdev_lvol.c:1589:_vbdev_lvol_set_xattr_cb: *ERROR*: Setting xattr failed: error <error_code>`

Reply:
It seems no convention in the error message.
https://github.com/longhorn/spdk/blob/43bee1800bf8c82d9b3d78605b21625ecf6742a3/module/bdev/lvol/vbdev_lvol.c#L1629